tests/test_api_paper.py

```diff
@@ -73,7 +73,7 @@ def test_buy(self):
 
     def test_sell_order_limit(self):
         directory = pathlib.Path(__file__).parent.resolve()
-        dummy = PaperBroker(str(directory) + "/../etc/dummy_account.yaml")
+        dummy = PaperBroker(str(directory) + "/../etc/dummy_account_copy.yaml")
```

Comment:
don't mess up the original dummy_account which should be used for yaml read validation.